src/components/Layout/SearchList.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
+
+import {
+  AggregatePlaceAttributes,
+  PlaceListOrderBy,
+} from "../../entities/Place/types"
+import watermelonIcon from "../../images/watermelon-icon.svg"
+import locations from "../../modules/locations"
+import { SegmentPlace } from "../../modules/segment"
+import NoResults from "./NoResults"
+import OverviewList from "./OverviewList"
+
+import "./SearchList.css"
+import "./OverviewList.css"
+
+export type SearchListProps = {
+  isLoadingPlaces: boolean
+  isLoadingWorlds: boolean
+  placeResultList?: AggregatePlaceAttributes[]
+  placeTotalResults?: number
+  worldResultList?: AggregatePlaceAttributes[]
+  worldTotalResults?: number
```

Comment:
should this be optionals? if one of the totals = undefined maybe `hasNoResults = false`?